testing/wcag/pa11yci.js

```diff
@@ -1,13 +1,83 @@
-const getComponentUrls = require('./get-component-urls');
+function componentUrl(urlPath) {
+  return `http://localhost:3000/rails/view_components/citizens_advice_components/${urlPath}`;
+}
 
 module.exports = {
   defaults: {
     chromeLaunchConfig: {
-      args: ['--no-sandbox']
+      args: ['--no-sandbox'],
     },
     ignore: [
-      'WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchId'       // ignore missing anchor links in design-sytem
-    ]
+      // ignore missing anchor links in design-sytem
+      'WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchId',
+    ],
   },
-  urls: getComponentUrls('http://localhost:6006'),
+  urls: [
```

Comment:
We could probably generate this list from the component previews if we wanted. Maybe the generate examples rake task. I don't mind this being a manual list though.